lib/server.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+const express = require('express');
+const app = express();
+const timestamp = require('../middleware/timestamp');
+const logger = require('../middleware/logger');
+
+app.use(express.json);
```

Comment:
should be express.json() - you're calling a function

this small mistake may be causing some huge problems :)